src/bundle/Resources/public/scss/_content-edit.scss

```diff
@@ -1,10 +1,18 @@
+.ez-content-edit {
+    h2,
```

Comment:
Please, avoid styling by tag selectors. Use CSS class selectors instead. In case of there's none, please add a new one.